docs/api/power-monitor.md

```diff
@@ -39,3 +39,24 @@ Emitted when the system changes to AC power.
 ### Event: 'on-battery' _Windows_
 
 Emitted when system changes to battery power.
+
+## Methods
+
+The `powerMonitor` module has the following methods:
+
+#### `powerMonitor.querySystemIdleState(idleThreshold, callback)`
+
+* `idleThreshold` Integer
+* `callback` Function
+  * `idleState` String - Can be `active`, `idle`, `locked` or `unknown`
+
+Calculate system idle state. `idleThreshold` is the amount of time (in seconds)
+before considered idle. State will be notified via callback represents current system
```

Comment:
There's something missing between "notified via callback" and "represents current system", I think

Reply:
it's wrapped line, full sentence is `State will be notified via callback represents current system
idle state.`.

Reply:
That still doesn't make any sense though. Would this be ok?

```diff
+ Calculate the system idle state. `idleThreshold` is the amount of time (in seconds)
+ before considered idle. `callback` will be called synchronously on some systems
+ and with an `idleState` argument that describes the system's state.
```

Reply:
I think so, it's probably same bucket for my English writing skill. I'll update as suggested.